study-group/README.md

```diff
@@ -0,0 +1,119 @@
+| [Swipe / Drag](#built-in-support)                   | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ✅ | ❌ |
+| [Intuitive Positioning](#built-in-support)          | ✅ | ✅ | ✅ | ❌ | ✅ | ✅ | ✅ | ❌ | ❌ | ❌ | ❌ | ✅ |
+| [User Interaction Trigger](#built-in-support)       | ✅ | ❌ | ❌ | ❌ | ❌ | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
+| [Mentions Accessibility](#policies)                 | ❌ | ✅ | ❌ | ✅ | ✅ | ✅ | ❌ | ✅ | ✅ | ❌ | ❌ | ❌ |
+| [Default Times Out](#policies)                      | ✅ | ✅ | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ✅ | ✅ | ✅ | ✅ |
+| [Allows Multiple](#policies)                        | ✅ | ✅ | ✅ | ✅ | ❌ | ❌ | ✅ | ✅ | ✅ | ❌ | ❌ | ❌ |
+| [Allows Duplicates](#policies)                      | ✅ | ✅ | ✅ | ❌ | ❌ | ❌ | ✅ | ❌ | ✅ | ❌ | ❌ | ❌ |
+| [Newest on Top](#policies)                          | ✅ | ✅ | ✅ | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
+| [Close Button Default](#policies)                   | ✅ | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
+| [Translucent](#policies)                            | ❌ | ❌ | ❌ | ✅ | ❌ | ❌ | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ |                                                                           
+
+
+
+## Features
+The list of features used in the [Table of Comparison](#table-of-comparison) comes from the major recurring patterns and ideas that showed up in my research of existing toast implementations. This list is by no means exhaustive, but it covers many of the core elements built into popular toast libraries. They approximately break down into three categories: Properties, Built-In Support, and Policies.
+
+### Properties
+Properties of a toast are specific, tangible parts of the object that a developer can set, toggle, or access. These are often the user-facing elements of the toast, and the general consensus of popular libraries regarding these properties is typically what sets toasts apart from other solutions like alerts and dialogs.
+
+- `title`: header text to be displayed above the main message of the toast. 
+
+- `message`: main text contents of the toast. 
+
+- `timeout`: time period (usually in ms) after which the toast disappears.
+
+- Close Button: dedicated button that closes the toast when clicked.
+
+- Call To Action: button(s) displayed on the toast prompting user engagement.
+
+- Callbacks: functions triggered when the toast enters or exits certain states.
+
+- Events: JavaScript events triggered when the toast enters or exits certain states.
+
+- `icon`: pictograph displayed with the toast.
+
+- Progress Bar: element on toast displaying time remaining before timeout.
+
+### Built-In Support
+The Built-In Support establishes how a toast is defined and used, as well as what it's allowed to be used with. It consists of a collection of implementation details, patterns, and technologies that are supported by popular libraries.
+
+- Theme Support: offers built-in theming options, such as `success`, `info`, or `error`.
```

Comment:
Does something capture custom theme support?

Reply:
That's a good question, the feature that best captures that is 'Global Configuration', which would allow you to create custom toast configurations and apply them to your toasts. Some of the libraries marked for this category used it global configs for creating custom themes, some used it for more general tweaks, but I figured they're similar enough to group them under the one field. I'll add a note on 'Global Configuration' mentioning it.

Reply:
Example:

[Notyf Global Configurations](https://github.com/caroso1222/notyf#global-configuration)
```js
const notyf = new Notyf({
  duration: 1000,
  types: [
    {
      type: 'warning',
      backgroundColor: 'orange',
      icon: {
        className: 'material-icons',
        tagName: 'i',
        text: 'warning'
      }
    },
    {
      type: 'info',
      backgroundColor: 'blue',
      icon: false
    },
    ...
  ]
});

...

notyf.open({
  type: 'info',
  message: 'Send us <b>an email</b> to get support'
});
```

Reply:
Changed name to Global Custom Configuration, moved the location in Features, and clarified language